packages/calypso-e2e/src/lib/flows/login-flow.ts

```diff
@@ -56,7 +56,7 @@ export class LoginFlow {
 	 */
 	async logIn(): Promise< void > {
 		await this.page.goto( getCalypsoURL( 'log-in' ) );
-		await this.baseflow();
+		await Promise.all( [ this.page.waitForNavigation(), this.baseflow() ] );
```

Comment:
Previously, the LoginFlow would complete once the enter button in the `/log-in` endpoint was hit. To successfully log in to WordPress.com, we need to actually wait until the dashboard is loaded. This `waitForNavigation` achieves that without relying on selectors.